src/PortingAssistantExtensionServer/Common/PALanguageServerConfiguration.cs

```diff
@@ -11,13 +11,15 @@ public static class PALanguageServerConfiguration
         private static string _extensionVersion { get; set; }
 
         private static string _visualStudioVersion { get; set; }
+        private static bool _enabledDefaultCredentials { get; set; }
 
         private static string _visualStudioFullVersion { get; set; }
 
         static PALanguageServerConfiguration()
         {
             _enabledContinuousAssessment = false;
             _enabledMetrics = false;
```

Comment:
We need the new metric field @aniruddhadave added in PA client that indicate customer use default provide or not.

Reply:
Added